src/enzo/EvolveLevel.C

```diff
@@ -842,6 +846,13 @@ int EvolveLevel(TopGridData *MetaData, LevelHierarchyEntry *LevelArray[],
     LCAPERF_STOP("CallPython");
 #endif
 
+#ifdef USE_LIBYT
+    LCAPERF_START("CallInSitulibyt");
+    //CallEmptyInteractivePython(LevelArray, MetaData, level, 0);
+    CallInSitulibyt(LevelArray, MetaData, level, 0);
+    LCAPERF_STOP("CallInSitulibyt");
+#endif
+
```

Comment:
Why is ``CallInSitulibyt`` called before flux correction? Would it be better to call it from ``EvolveHierarchy``? It's already being called after ``EvolveLevel``'s recursive loop.

Reply:
I believe I did this exclusively to mimic the behavior of the existing python in situ stuff.  I think as long as it's able to be called from either the top or bottom of the W cycle it's fine to call it.

Reply:
I'm not entirely sure, nor which would be more desired.  I think I was attempting to replicate as best I could the interpolated output machinery.